man/man5/zfs-module-parameters.5

```diff
@@ -1752,14 +1769,97 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Max concurrent I/Os per top-level vdev (mirrors or raidz arrays) allowed during
-scrub or resilver operations.
+legacy scrub or resilver operations.
 .sp
 Default value: \fB32\fR.
 .RE
 
 .sp
 .ne 2
 .na
+\fBzfs_scan_md_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os allowed during scrub or resilver metadata scanning phase.
+.sp
+Default value: \fB1024\fR.
+.RE
+
+.sp
+.ne 2
+.na
+\fBzfs_scan_leaf_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os per leaf vdev allowed during scrub or resilver data
+verification phase.
+.sp
+Default value: \fB50\fR.
```

Comment:
That makes sense. I guess I hadn't considered that a scrub IO will issue an IO to each disk anyway. I can combine these.